qiita_db/metadata_template.py

```diff
@@ -1051,6 +1085,27 @@ def get_filepaths(self, conn_handler=None):
 
         return [(fpid, base_fp(fp)) for fpid, fp in filepath_ids]
 
+    def categories(self):
+        """Get the categories associated with self
+
+        Returns
+        -------
+        set
```

Comment:
The order of columns does not matter with the exception of the index (SampleID). The test for equality should respect this.